### PR TITLE
Update setup and build instructions in README

### PR DESCRIPTION
**Pull request by RickyC0626**
Help developers set up pre-commit workflow, virtual environment, and necessary dependencies in order to run `pytest` successfully.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,7 +72,15 @@ pip install manimpango --no-binary :all:
 
 **Note**: `pip` by default uses wheels, so make sure to pass the `--no-binary` parameter.
 
-##### Using `git` clones / Contributing
+#### Using `git` clones / Contributing
+
+Please remember to do this inside your virtual environment, if you want to use your **Manimpango** with **Manim**.
+
+```sh
+pip install virtualenv
```
Comment (naveen521kk) [inline README.md:80]: seems unnecessary
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -63,7 +72,15 @@ pip install manimpango --no-binary :all:
...
+python -m venv ./venv
+source venv/bin/activate
```
Comment (naveen521kk) [inline README.md:82]: this is different for windows?

File: README.md [hunk 3]
```diff
@@ -74,12 +91,23 @@ pip install Cython
 After that you can use `pip` to install the clone with the following command:
 
 ```sh
-pip install .
+pip install -r requirements-dev.txt .
```
Comment (naveen521kk) [inline README.md:94]: They need to run both for installing ManimPango itself.
```suggestion
pip install -e .
pip install -r requirements-dev.txt .
```

File: README.md [hunk 4]
```diff
@@ -74,12 +91,23 @@ pip install Cython
...
 ```
 
-You will need to this way if you want to *contribute* to **ManimPango**.
+Next, run the setup script:
+
+```sh
+python setup.py install
+python setup.py build_ext -i
```
Comment (naveen521kk) [inline README.md:101]: no need to do `setup.py` install.
```suggestion
python setup.py build_ext -i
```

**Review by naveen521kk — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (RickyC0626) [reply README.md]: Added the command for windows

**Review by naveen521kk — APPROVED** [review]

**Pull request merged**